### DISCUSSION TITLE
Poll amount of shares periodically

### DISCUSSION THREAD

**Discussion by sebastianberm** [jobvk/Home-Assistant-Windcentrale#71] [Ideas]
At the moment, I have been working on expanding the amount of shares I have. However, it seems, like the API request to retrieve the amount of shares per mill is only done once, at the initial login?

It might be an idea to for example do this once per week or so or provide a button in the config to poll this again. For now, to fix my immediate 'issue', I have modified the core.config_entries file to match the amount Windcentrale displays.

Due to the fact it does not interfere with the current functions, I don't really see this as a bug. However, a (better) way to deal with this, would actually be nice.

**Comment by jobvk** [comment]
@sebastianberm great idea💡,

I Think a button is for sure possible. Currently I'm travelling for the Holidays but from the start of the new year I'll start building on the button.

**Comment by jobvk** [comment]
Idea 💡 was added in Release [2023.10.0](https://github.com/jobvk/Home-Assistant-Windcentrale/releases/tag/2023.10.0)
